sources/bloodhound/src/bin/bottlerocket-checks/checks.rs

```diff
@@ -0,0 +1,935 @@
+use bloodhound::results;
+use bloodhound::*;
+use std::os::unix::fs::PermissionsExt;
+use std::process::Command;
+use walkdir::WalkDir;
+
+const PROC_MODULES_FILE: &str = "/proc/modules";
+const PROC_CMDLINE_FILE: &str = "/proc/cmdline";
+const LOCKDOWN_FILE: &str = "/sys/kernel/security/lockdown";
+const CHRONY_CONF_FILE: &str = "/etc/chrony.conf";
+const JOURNALD_CONF_FILE: &str = "/usr/lib/systemd/journald.conf.d/journald.conf";
+const SYSCTL_CMD: &str = "/usr/sbin/sysctl";
+const SYSTEMCTL_CMD: &str = "/usr/bin/systemctl";
+const MODPROBE_CMD: &str = "/bin/modprobe";
+const SESTATUS_CMD: &str = "/usr/bin/sestatus";
+const IPTABLES_CMD: &str = "/usr/sbin/iptables";
+const IP6TABLES_CMD: &str = "/usr/sbin/ip6tables";
+
+// =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<=
+
+pub struct BR01010101Checker {}
+
+impl results::Checker for BR01010101Checker {
+    fn execute(&self) -> results::CheckerResult {
+        let mut result = results::CheckerResult {
+            error: String::new(),
+            status: results::CheckStatus::SKIP,
+        };
+
+        check_udf_loaded(&mut result);
+        check_modprobe(&mut result);
+
+        result
+    }
+
+    fn metadata(&self) -> results::CheckerMetadata {
+        results::CheckerMetadata {
+            title: "Ensure mounting of udf filesystems is disabled".to_string(),
+            id: "1.1.1.1".to_string(),
+            level: 2,
+            name: "br01010101".to_string(),
+            mode: results::Mode::Automatic,
+        }
+    }
+}
+
+/// Parses the loaded modules to see if the udf module is already loaded.
+fn check_udf_loaded(result: &mut results::CheckerResult) {
+    if result.status == results::CheckStatus::FAIL {
+        return;
+    }
```

Comment:
nit: it would be more consistent with other checkers to do this in the execute method for `BR01010101Checker`